osvr/RenderKit/RenderManagerOpenGL.cpp

```diff
@@ -934,12 +956,16 @@ namespace renderkit {
             // If makeCurrent() fails give up on destroying OpenGL objects
             return;
         }
-#ifndef OSVR_RM_USE_OPENGLES20
+
         if (VAO) {
+#ifdef OSVR_RM_USE_OPENGLES20
```

Comment:
Not all implementations of OPENGLES 2.0 have the extension.  In particular, the Ubuntu Tegra implementation is missing it.

Reply:
Looks like maybe a m_GLVAOExtensionAvailable is being set that should be used here to guard this call?  Except that won't help with linking failure.

Reply:
The GL procs are dynamically loaded at runtime, so there shouldn't be any linking failure. The if(VAO) check is sufficient as a guard here, as VAO is only initialized when m_GLVAOExtensionAvailable is true, are we're testing both successful VAO initialization and VAO extension available in one expression.

Reply:
Cool.